arangod/Replication2/Methods.cpp

```diff
@@ -632,9 +630,9 @@ struct ReplicatedLogMethodsCoordinator final
           std::vector<LogIndex> indexes;
           indexes.reserve(payloadSize);
           auto indexIter = velocypack::ArrayIterator(result.get("indexes"));
-          std::transform(
-              indexIter.begin(), indexIter.end(), std::back_inserter(indexes),
-              [](auto const& it) { return it.template extract<LogIndex>(); });
+          for (auto it : indexIter) {
+            indexes.push_back(it.template extract<LogIndex>());
+          }
```

Comment:
WTF?
Why would you do that? `std::transform` does autovectorizationand other cool stuff. Like AVX2 on nummeric transformations etc. Loop unrolling. Why?

Reply:
std::transform never do auto vectorization for Object/ArrayIterator, because they are forward iterator.

And btw if it's auto vectorization it's same as for loop auto vectorization

Reply:
It was done because std::transform not supported different types of iterators